web-console/src/views/datasource-view/datasource-view.tsx

```diff
@@ -151,7 +151,7 @@ export class DatasourcesView extends React.Component<DatasourcesViewProps, Datas
 
         const seen = countBy(datasources, (x: any) => x.datasource);
 
-        const disabledResp = await axios.get('/druid/coordinator/v1/metadata/datasources?includeDisabled');
+        const disabledResp = await axios.get(this.state.showDisabled ? '/druid/coordinator/v1/metadata/datasources?includeDisabled' : '/druid/coordinator/v1/metadata/datasources' );
```

Comment:
if you do not need to get the disabled datasources then why make this query at all?